### ISSUE TITLE
AWMS can be re-paramed

### ISSUE THREAD

**Issue by splinter21** [ChaofWang/AWSRN#17]
inp=torch.randn(1,16,8,8)
awms=AWMS()#去掉wn的版本
opt1=awms(inp)
k3=awms.tail_k3.weight
b3=awms.tail_k3.bias
k5=awms.tail_k5.weight
b5=awms.tail_k5.bias
k7=awms.tail_k7.weight
b7=awms.tail_k7.bias
k9=awms.tail_k9.weight
b9=awms.tail_k9.bias
scale_k3=awms.scale_k3.scale
scale_k5=awms.scale_k5.scale
scale_k7=awms.scale_k7.scale
scale_k9=awms.scale_k9.scale
conv9r=nn.Conv2d(in_channels=16, out_channels=3, kernel_size=9, stride=1,padding=4, bias=True)
conv9r.weight.data=scale_k3 * torch.nn.functional.pad(k3, [3, 3, 3, 3])+scale_k5 * torch.nn.functional.pad(k5, [2, 2, 2, 2])+scale_k7 * torch.nn.functional.pad(k7, [1, 1, 1, 1])+scale_k9 * k9
conv9r.bias.data=scale_k3 * b3+scale_k5 * b5+scale_k7 * b7+scale_k9 * b9
opt2=awms.pixelshuffle(conv9r(inp))
delta = opt2 - opt1
print(np.abs(delta.detach().cpu().numpy()).mean())

输出1.13682596e-07

4个卷积4次pixelshuffle变成1个9x9卷积1次pixelshuffle
计算量9 * 9/(3 * 3+5 * 5+7 * 7+9 * 9)约为一半
至于是否wn的转化可参考weight_norm.py的代码进行卷积参数的互相转化

**Comment by ChaofWang** [comment]
非常感谢提醒！这部分确实适合通过重参数化进行合并。
